x/ibc-switch/types/keys.go

```diff
@@ -0,0 +1,21 @@
+package types
+
+import "strings"
+
+const (
+	ModuleName = "ibc-switch"
+)
+
+// RouterKey is the message route. Can only contain
+// alphanumeric characters.
+var RouterKey = strings.ReplaceAll(ModuleName, "-", "")
+
+var (
+	KeySwitchStatus  = []byte("switchstatus")
+	KeyPauserAddress = []byte("pauseraddress")
+)
+
+const (
+	IbcSwitchStatusOff string = "off"
+	IbcSwitchStatusOn  string = "on"
```

Comment:
I thought this one would be more direct since you can't go wrong if it's "off", but if it was 0 you wouldn't know if the switch lets packets through or doesn't. But maybe you're right. Although maybe not essential since I would have to change all the messages in secretjs?